source/_integrations/bosch_alarm.markdown

```diff
@@ -63,3 +63,8 @@ Since the _Mode 2_ automation user has "superuser" privileges, it bypasses the r
 This integration follows standard integration removal. No extra steps are required.
 
 {% include integrations/remove_device_service.md %}
+
+### Diagnostics information
```

Comment:
I would move this above the removal procedure and nest it under  ## Troubleshooting
I think the removal procedure should be at the end of the page as it marks the end of the lifecycle of an integration on a given installation.